packages/apps/job-launcher/server/src/common/pipes/query.ts

```diff
@@ -0,0 +1,20 @@
+import { ArgumentMetadata, Injectable, PipeTransform } from '@nestjs/common';
+import { camelCase } from 'lodash';
+
+@Injectable()
+export class NormalizeQueryParamsPipe implements PipeTransform {
```

Comment:
we already have [this](https://github.com/humanprotocol/human-protocol/blob/9ef1109770de65ddacb84f422c326078f4aa784a/packages/apps/job-launcher/server/src/common/interceptors/snake-case.ts#L12) interceptor